packages/web3-core-method/src/methods/admin/AdminAddPeerMethod.js

```diff
@@ -0,0 +1,49 @@
+/*
+    This file is part of web3.js.
+
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file AdminAddPeerMethod.js
+ * @author Prince Sinha <sinhaprince013@gmail.com>
+ * @date 2019
+ */
+
+import AbstractMethod from '../../../lib/methods/AbstractMethod';
+
+export default class AdminAddPeerMethod extends AbstractMethod {
+    /**
+     * @param {Utils} utils
+     * @param {Object} formatters
+     * @param {AbstractWeb3Module} moduleInstance
+     *
+     * @constructor
+     */
+    constructor(utils, formatters, moduleInstance) {
+        super('admin_addPeer', 1, utils, formatters, moduleInstance);
+    }
+
+    /**
+     * This method will be executed after the RPC request.
+     *
+     * @method afterExecution
+     *
+     * @param {Object} response
+     *
+     * @returns {Array}
+     */
+    afterExecution(response) {
```

Comment:
The method ``afterExecution`` has only to be implemented if you would like to format the response. :)

Reply:
Yes got it. Done @nivida